config/config.go

```diff
@@ -799,16 +799,21 @@ func (c *Config) CheckExchangeConfigValues() error {
 				return fmt.Errorf(ErrExchangeBaseCurrenciesEmpty, c.Exchanges[i].Name)
 			}
 			if c.Exchanges[i].AuthenticatedAPISupport { // non-fatal error
-				if c.Exchanges[i].APIKey == "" || c.Exchanges[i].APISecret == "" ||
-					c.Exchanges[i].APIKey == DefaultUnsetAPIKey ||
-					c.Exchanges[i].APISecret == DefaultUnsetAPISecret {
+				if c.Exchanges[i].APIKey == "" || c.Exchanges[i].APIKey == DefaultUnsetAPIKey {
+					c.Exchanges[i].AuthenticatedAPISupport = false
+					log.Warnf(WarningExchangeAuthAPIDefaultOrEmptyValues, c.Exchanges[i].Name)
+				}
+
+				if (c.Exchanges[i].APISecret == "" || c.Exchanges[i].APISecret == DefaultUnsetAPISecret) &&
+					c.Exchanges[i].AuthenticatedAPISupport && c.Exchanges[i].Name != "COINUT" {
+					c.Exchanges[i].AuthenticatedAPISupport = false
+					log.Warnf(WarningExchangeAuthAPIDefaultOrEmptyValues, c.Exchanges[i].Name)
+				}
+
+				if (c.Exchanges[i].ClientID == "ClientID" || c.Exchanges[i].ClientID == "") && c.Exchanges[i].AuthenticatedAPISupport &&
```

Comment:
Same for here, then this looks GTG!

Reply:
Main reason was to prevent multiple log.Warns at the end. However, I'll remove those extra checks and just add a 
```
if !c.Exchanges[i].AuthenticatedAPISupport {
					log.Warnf(WarningExchangeAuthAPIDefaultOrEmptyValues, c.Exchanges[i].Name)
				}
```
at the end

Reply:
Sounds good 👍